src/routes/Dashboard/SearchPlateForm.jsx

```diff
@@ -1,16 +1,20 @@
 import {useState} from "react";
 import Button from "components/main/Button";
+import PlateDeleteBtn from "components/main/PlateDeleteBtn";
 
-export default function SearchPlateForm() {
+export default function SearchPlateForm({onSearch, onClear}) {
```

Comment:
missing `PropTypes` at the end of this file to define `onSearch` and `onClear` props